Department.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
+class Department
+{
+    public function __construct(array $employees, string $name = "Apelsin")
+    {
+        $this->employees = $employees;
```

Comment:
Поля класса `$employees` и `$name` должны быть объявлены выше. Или можно сразу сделать так:
```suggestion
        public function __construct(private array $employees, private string $name = "Apelsin")
        {
        }
```